cmd/server/main.go

```diff
@@ -1,3 +1,40 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"github.com/ilya372317/must-have-metrics/internal/config"
+	"github.com/ilya372317/must-have-metrics/internal/constant"
+	"github.com/ilya372317/must-have-metrics/internal/router"
+	"github.com/ilya372317/must-have-metrics/internal/storage"
+	"log"
+	"net/http"
+)
+
+var (
+	repository storage.AlertStorage
+	host       *string
+)
+
+func init() {
+	repository = storage.MakeInMemoryStorage()
+	cnfg := new(config.ServerConfig)
+	if err := cnfg.Init(); err != nil {
+		log.Fatalln(err.Error())
+	}
+	host = flag.String("a", "localhost:8080", "server address")
```

Comment:
```suggestion
	host = flag.String("a", DEFAULT_SERVER_ADDR, "server address")
```
было бы удобней вынести в константу

Reply:
++